src/printapp/templates/index.html

```diff
@@ -451,10 +433,12 @@ <h6 class="bold">Laser Printer</h6>
           </tr>
         </tbody>
       </table>
+	</div>
+	</div>
     </div>
   </div>
 
-  <hr />
+      <hr />
```

Comment:
You guys are going to have to run this through your own formatters.  My IDEs don't really like the templates in <script> tags and I don't really want to adjust my settings to do 2-space indents.